sampleApp/sample.tsx

```diff
@@ -90,8 +111,15 @@ oneNoteDataProvider.getNotebooks().then((notebooks) => {
 			ariaSelectedId: initialSelectedId
 		}
 	};
+	let notebookName = findNotebook(notebooks, initialSelectedId)
 	render(globalProps, notebooks);
+	renderDropdown(globalProps, notebooks, notebookName, false);
 }).catch((value) => {
 	// tslint:disable-next-line:no-console
 	console.error(value);
 });
+
+export function findNotebook(notebooks, itemid) {
```

Comment:
The itemId may not represent a notebook, it could be anything. We should rename this function.